src/LegacyElementPortal.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import getPropTypes from './utils/getPropTypes';
+import renderToNodes from './utils/renderToNodes';
+
+const ElementPortal = React.createClass({
```

Comment:
Could we have a comment here explaining that we consider this `Legacy` because it's for React < 16 support? Just so it's clear for whoever gets to this file in the future!

Reply:
👍